_data/fr/navigation.yml

```diff
@@ -0,0 +1,28 @@
+nav:
+- title:                 Utiliser Jamulus
+  subfolderitems:
+    - page:              Pour commencer
+      url:               /wiki/Getting-Started
+      subsubfolderitems:
+        - page:          Windows
+          url:           /wiki/Installation-for-Windows
+        - page:          Macintosh
+          url:           /wiki/Installation-for-Macintosh
+        - page:          GNU/Linux
+          url:           /wiki/Installation-for-Linux
+    - page:              Manuel du logiciel
+      url:               /wiki/Software-Manual
+    - page:              Problèmes potentiels
+      url:               /wiki/Client-Troubleshooting
+- title:                 Plus
+  subfolderitems:
+    - page:              Installer un serveur
+      url:               /wiki/Running-a-Server
+    - page:              Configuration
```

Comment:
Always according to the French UI of the software, we should use « Paramètres »
(screenshots for further use)

<img width="226" alt="Capture d’écran 2020-11-19 à 07 49 23" src="https://user-images.githubusercontent.com/9108457/99632735-fc591400-2a3d-11eb-9728-25910c49c137.png">

<img width="780" alt="Capture d’écran 2020-11-19 à 07 48 45" src="https://user-images.githubusercontent.com/9108457/99632786-1bf03c80-2a3e-11eb-8b28-b8eb43ff55f6.png">